Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine as builder
+FROM golang:1.20-alpine as builder
```

Comment:
Nice! Please update the destination version of Alpine to 3.17 (~line 7).

Reply:
Swapped out this one to golang:1.20-alpine3.17 to make it more obvious there is correlation with the second.